sdk/src/main/scala/com/horizen/certificatesubmitter/CertificateSubmitter.scala

```diff
@@ -564,7 +571,20 @@ class CertificateSubmitter(settings: SidechainSettings,
       signersPublicKeyWithSignatures)
   }
 
+  private def lastSidechainEpochNumberForWithdrawalEpochNumber(history: SidechainHistory, withdrawalEpochNumber: Int): ConsensusEpochNumber = {
```

Comment:
I think it's better to name it last**Consensus**EpochNumberForWithdrawalEpochNumber.